Project.toml

```diff
@@ -29,6 +29,7 @@ AssetRegistry = "0.1"
 Cassette = "≥ 0.2.5"
 GeometryTypes = "0.6, 0.7"
 JSExpr = "0.3, 0.5"
+MsgPack = "≥ 1.0.0"
```

Comment:
Let's do `MsgPack = "1"` so we don't assume compatibility with MsgPack 2.0 and above.

Reply:
Unique", integers that you could think as a good solution of Z.
Namely,-1:
p*s=606815585
-606815585,s-1
-121363117,s-5
-3722795,s-163
-74455i,?,-815*s?
-815,s=-?dS
-163,-372279
972273-,361-
Sd?-=s,518-
?s*518-,?,i55447-
361-s,5972273-
5-s,711363121-
1-s,585518606-
585518606=s*p
:1-,ylemaN
.Z fo noitulos doog a sa kniht dluoc uoy taht sregetni ,"euqinU

Things start to get weird when you flip this as in base hex and eval at base 64x and append to its complex conjugate of any other family of bases.